opds.py

```diff
@@ -123,13 +123,15 @@ def slugify_feed_title(cls, feed_title):
         slug = re.sub(' {2,}', ' ', slug)
         return unicode('-'.join(slug.split(' ')))
 
-    def __init__(self, base_url, lane=None, include_search=None, prefix=None):
+    def __init__(self, base_url, lane=None, prefix=None, include_search=None,
+                 license_link=None, suppressed_covers=None):
         if not base_url.endswith('/'):
```

Comment:
It has more to do with my mental model than anything else, but it won't break any of annotator calls because there's no implicit ordering of keyword arguments in the script.